common-api/src/main/java/org/pmiops/workbench/cdr/model/Concept.java

```diff
@@ -32,6 +32,7 @@
     private float prevalence;
     private List<String> synonyms = new ArrayList<>();
     private String synonymsStr;
+    private String path;
```

Comment:
This is something new. 
I have copied the path of concept from the achilles_results to here. 
Thought this would be easy to prep the data for ui.